python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -745,6 +746,12 @@ def quantize(model: nn.Module,
                                only valid when accelerator='onnxruntime' or accelerator='openvino',
                                otherwise will be ignored. If output_tensors=False, output of the
                                export model will be ndarray.
+        :param example_kwarg_inputs: a pack of keyword arguments of example inputs that will be
```

Comment:
Why this comment is different from others

Reply:
all ipex models have the same comment.